Randomizer.py

```diff
@@ -0,0 +1,24 @@
+import random
+
+
+def randomly(list_of_students):
+    student_dict_reverted = {}
+    for elem in list_of_students:
+        student_dict_reverted[elem] = 0
+    print('Do you want to make your choice?', '\n', 'Press y to do it, n to exit')
+    while True:
+        choice2 = input()
+        if choice2 == 'n':
+            break
+        elif choice2 == 'y':
+            somestud = random.choice(list_of_students)
+            if student_dict_reverted[somestud] > 0:
+                somestud = random.choice(list_of_students)
+                student_dict_reverted[somestud] = 0
+                print(somestud)

```

Comment:
Repeat of prints and `student_dict_reverted[somestud] += 1` can be avoided by moving them outside condition statement

Reply:
Did not notice this.
Thanks, corrected.